internal/cmd/connect/command_plugin_install.go

```diff
@@ -0,0 +1,456 @@
+package connect
+
+import (
+	"archive/zip"
+	"bytes"
+	"crypto/md5"
+	"crypto/sha1"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/form"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/utils"
+)
+
+type manifest struct {
+	Owner struct {
+		Username string `json:"username"`
+		Name     string `json:"name"`
+	} `json:"owner"`
+	Name    string `json:"name"`
+	Title   string `json:"title"`
+	Version string `json:"version"`
+	Archive struct {
+		Url  string `json:"url"`
+		Md5  string `json:"md5"`
+		Sha1 string `json:"sha1"`
+	} `json:"archive"`
+	License []struct {
+		Name string `json:"name"`
+		Url  string `json:"url"`
+	} `json:"license"`
+}
+
+const (
+	archiveInstallation = "ARCHIVE"
+	packageInstallation = "PACKAGE"
+
+	invalidDirectoryErrorMsg       = `plugin directory "%s" does not exist`
+	unexpectedInstallationErrorMsg = "unexpected installation type: %s"
+)
```

Comment:
Use a new `const` block when grouping related constants!
```suggestion
const (
	archiveInstallation = "ARCHIVE"
	packageInstallation = "PACKAGE"
)

const (
	invalidDirectoryErrorMsg       = `plugin directory "%s" does not exist`
	unexpectedInstallationErrorMsg = "unexpected installation type: %s"
)
```

Reply:
(Do we really need the first two, though?)

Reply:
Nah not really, I just replaced references to these consts with the actual string